src/api/route-services/projects.js

```diff
@@ -243,6 +243,7 @@ class ProjectsService {
         const deletePromises = experiments.reduce((acc, experimentId) => {
           acc.push(experimentService.deleteExperiment(experimentId));
           acc.push(samplesService.deleteSamplesEntry(projectUuid, experimentId, sampleUuids));
```

Comment:
I can't see a unit test to check that when we call `deleteProject` in `src/api/route-services/projects.js`, then the access gets deleted too. Could you modify the deleteProject test that we have right now to check for that?